nautobot/core/templates/inc/nav_menu.html

```diff
@@ -13,795 +13,52 @@
                 <img src="{% static 'img/nautobot_logo.svg' %}" height="30" />
             </a>
         </div>
+
         <div id="navbar" class="navbar-collapse collapse">
             {% if request.user.is_authenticated %}
```

Comment:
I noticed in one of our customer deployments that's set up to allow unauthenticated users, that while the entire home screen is visible, the menu bar is completely empty due to this check. I'm not sure that 9994ae02b did the right thing here. Might be worth some investigation.

Reply:
Maybe just adding `or not settings.HIDE_RESTRICTED_UI` would be an appropriate fix here?